tests/Microsoft.Fx.Portability.Tests/ApiPortServiceTests.cs

```diff
@@ -15,5 +24,45 @@ public void VerifyParameterChecks()
             Assert.Throws<ArgumentOutOfRangeException>(() => new ApiPortService(string.Empty, new ProductInformation("")));
             Assert.Throws<ArgumentOutOfRangeException>(() => new ApiPortService(" \t", new ProductInformation("")));
         }
+
+        [Fact(Skip = SkipReason)]
+        public void ApiPortService_GetDocIdsWithValidDocId()
+        {
+            var apiPortService = new ApiPortService(
+                ServiceEndpoint,
+                new ProductInformation("ApiPort_Tests", typeof(ApiPortServiceTests)));
+
+            var docIds = new List<string>
+            {
+                "T:System.Console",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Object,System.Xml.Serialization.XmlSerializationWriter)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
+                "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces,System.String)"
+            };
+
+            var result = apiPortService.GetDocIds(docIds).Result.Response;
+            Assert.Equal(docIds.Count(), result.Count());
+            Assert.Equal(0, docIds.Except(result.Select(r => r.Definition.DocId)).Count());
```

Comment:
I forgot about using that LINQ trick to check that two collections are equal. :+1: